web/scripts/ui.js

```diff
@@ -410,6 +413,13 @@ export class ComfyUI {
 			this.history.update();
 		});
 
+		const confirmClear = this.settings.addSetting({
+			id: "Comfy.ConfirmClear",
+			name: "Require confirmation when clearing workflow",
+			type: "boolean",
+			defaultValue: true,
```

Comment:
On a clean browser I see the checkbox ticked but it doesn't respect this value because it doesn't ask for confirmation.

Reply:
Added missing default value